pipeline/mw-pipeline-v0.1.yaml

```diff
@@ -244,7 +208,7 @@ spec:
             type: string
         steps:
           - name: notification
-            image: $(params.PNC_KONFLUX_TOOLING_IMAGE)
+            image: quay.io/konflux-ci/pnc-konflux-tooling:latest@sha256:8c1c50b01c8dc5e3847f2ace4da8d28d255935b15c14d043bb2e819941697b29
```

Comment:
This repo might have to be onboarded to konflux unless renovate can update the image references.